frontend/src/app/accountCreation/MfaEmailVerify/MfaEmailVerify.stories.tsx

```diff
@@ -7,7 +7,7 @@ export default {
   component: MfaEmailVerify,
   argTypes: {},
   args: {
-    location: { state: { email: "foo@bar.com" } },
+    location: { state: { contact: "foo@bar.com" } },
```

Comment:
Changed this to `contact` since it is now using the generalized `MfaVerify` component